Resources/Prototypes/Entities/Objects/Specific/Salvage/ore_bag.yml

```
@@ -26,3 +26,20 @@
         - ArtifactFragment
         - Ore
   - type: Dumpable
+
+- type: entity
+  name: internal ore bag
+  id: BorgOreBag
+  parent: OreBag
+  description: A large ore bag built into the frame of mining cyborgs.
+  components:
+    - type: Storage
+      maxItemSize: Normal
+      grid:
+      - 0,0,9,5
+      quickInsert: true
+      areaInsert: true
+      whitelist:
+        tags:
+          - ArtifactFragment
+          - Ore 
```

A lot of these are inherited. You don't need to put all of this, only the stuff you changed.
```suggestion
  components:
    - type: Storage
      grid:
      - 0,0,9,5
```